%YAML:1.0

folder: /home/milo/datasets/Unity3D/farmsim/waypoints1
playback_speed: 1.0
